src/api/interactivity.js

```diff
@@ -65,6 +108,25 @@ export default class Interactivity {
         const nwmc = wToR(wm.x, wm.y, { scale: WM_R, center: { x: 0, y: 0 } });
         return [].concat(...this._layerList.map(layer => layer.getFeaturesAtPosition(nwmc)));
     }
+    
+    /**
+     * Return the difference between the feature arrays A and B.
+     * The output value is also an array of features.
+     */
+    _getDiffFeatures(featuresA, featuresB) {
+        let featuresDiff = [];
+        const IDs = this._getFeatureIDs(featuresB);
+        for (let feature of featuresA) {
```

Comment:
I think this can be simplified by a `filter()`

Reply:
Yep, of course!